wpilibc/Athena/src/PWM.cpp

```diff
@@ -43,7 +43,7 @@ PWM::PWM(uint32_t channel) {
 
   m_channel = channel;
 
-  setPWM(m_pwm_ports[m_channel], kPwmDisabled, &status);
+  setPWMRaw(m_pwm_ports[m_channel], kPwmDisabled, &status);
```

Comment:
I know you didn't do this, but why is there an array of pwm ports? This doesn't exist on the Java side.